pkg/virt-api/webhooks/validating-webhook/admitters/validate-k8s-utils.go

```diff
@@ -0,0 +1,258 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+Copyright 2022 The KubeVirt Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+
+The code here is taken from https://github.com/kubernetes/kubernetes/blob/29fb8e8b5a41b2a7d760190284bae7f2829312d3/pkg/apis/core/validation/validation.go#L3288
+all the "core" package types are converted to corev1 types in order to avoid vendoring in k8s.
+
+https://github.com/kubernetes/kubernetes/blame/29fb8e8b5a41b2a7d760190284bae7f2829312d3/pkg/apis/core/validation/validation.go#L3288
+the code hardly changes all of the changes have been atleast a few years older
+this makes it easier to copy and maintain instead of vendoring in kubernetes or
+creating dry runs of the pod object during admission validation.
+*/
+
+package admitters
+
+import (
+	corev1 "k8s.io/api/core/v1"
```

Comment:
how about just changing this import to

```
core "k8s.io/api/core/v1"
```

would that reduce the changes to just changing the import and leave the rest as-is?

Reply:
@rmohr great idea, it will reduce the number of changes in copy paste, will do it.

Reply:
updated, PTAL @rmohr